README.md

```diff
@@ -48,30 +48,27 @@ This provides a web UI for AI chat playground that is able to connect virtually
     $REPOSITORY_ROOT = git rev-parse --show-toplevel
     ```
 
-1. Run the app.
+1. Add Github token for Github Models connection.
 
     ```bash
-    dotnet run --project $REPOSITORY_ROOT/src/OpenChat.AppHost
+    # bash/zsh
+    dotnet user-secrets --project $REPOSITORY_ROOT/src/OpenChat.PlaygroundApp \
+        set GitHubModels:Token "YOUR-TOKEN"
     ```
 
-1. If you want to change the model, pass the following arguments:
-
     ```bash
-    dotnet run --project $REPOSITORY_ROOT/src/OpenChat.AppHost -- [OPTIONS]
+    # PowerShell
+    dotnet user-secrets --project $REPOSITORY_ROOT/src/OpenChat.PlaygroundApp `
+        set GitHubModels:Token "YOUR-TOKEN"
     ```
+    > The Token needs "Models" "read-only" permission.
+    > For more detailed information dealing tokens, refer to [Managing-your-personal-access-tokens](https://docs.github.com/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).

```

Comment:
This should be like:

`refer to the doc, [Managing your personal access tokens](https://docs.github.com/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).`